t_acc/test_read_remap_file/test_read_remap_file.c

```diff
@@ -0,0 +1,413 @@
+#include <openacc.h>
```

Comment:
This file is missing the license header. Also need documentation describing the test being performed and the expected outcomes.